src/api/api.ts

```diff
@@ -0,0 +1,7 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
In this project, I noticed an excessive repetition of the BASE_URL variable. It’s a good idea to avoid repeating this variable in multiple places. I recommend moving it to a centralized constant or configuration to facilitate maintenance and reuse of the variable.